eng/targets/Helix.targets

```diff
@@ -34,11 +34,11 @@
     <HelixPreCommand Include="call RunPowershell.cmd InstallNode.ps1 $(NodeVersion) || exit /b 1" />
   </ItemGroup>
 
-  <ItemGroup Condition=" '$(TestDependsOnAspNetRuntime)' == 'true' ">
-    <!-- Grab all shipping packages. -->
-    <HelixContent Include="$(RepoRoot)artifacts\packages\$(Configuration)\Shipping\*$(SharedFxVersion).nupkg" />
+  <ItemGroup Condition=" '$(TestDependsOnTemplates)' == 'true' ">
+    <!-- Grab all shipping template packages. -->
+    <HelixContent Include="$(RepoRoot)artifacts\packages\$(Configuration)\Shipping\*Templates*$(SharedFxVersion).nupkg" />
```

Comment:
Doesn't building templated projects require the just-built packages for everything else❔ I expected this to keep grabbing all `*.nupkg` files except Microsoft.AspNetCore.App.*.

Reply:
Oops right, I forgot we need them all for the feed, this actually makes things easier anyways

Reply:
I'll renamed it to TestDependsOnAspNetPackages to copy over all the nupkgs, Interop/Templates/UnitTests will be the only ones that enable this